client/src/document/organisms/toc/index.tsx

```diff
@@ -5,8 +5,23 @@ import { Toc } from "../../../../../libs/types/document";
 import { useFirstVisibleElement } from "../../hooks";
 import { useGleanClick } from "../../../telemetry/glean-context";
 import { TOC_CLICK } from "../../../telemetry/constants";
+import { useLocale } from "../../../hooks";
+import { DEFAULT_LOCALE } from "../../../../../libs/constants";
+
+const DEFAULT_TITLE = {
+  "en-US": "In this article",
+  es: "En este artículo",
+  fr: "Dans cet article",
+  ja: "この記事では",
+  ko: "이 문서에서는",
```

Comment:
@mdn/yari-content-ko Would you mind taking a look if this translation is correct?

PS: Please note that this is a one-off UI localization, and currently we do not accept any community PRs for UI localization.

Reply:
<img width="148" alt="image" src="https://github.com/mdn/yari/assets/23524849/817ff20a-e39f-4e32-9b9f-7aa72ca4435f">

@mdn/yari-content-ko  "목차"로 하는게 어떨까요?

Reply:
@1ilsang @hochan222 Do you want to change it as follows:

```suggestion
  ko: "목차"로 하는게 어떨까요",
```

Reply:
I agree :)